membership-attribute-service/app/repositories/MembershipAttributesSerializer.scala

```diff
@@ -24,13 +24,13 @@ case class MembershipAttributesSerializer(tableName: String)
     Map(
       AttributeNames.userId -> membershipAttributes.UserId,
       AttributeNames.membershipNumber -> membershipAttributes.MembershipNumber.getOrElse(""),
-      AttributeNames.tier -> membershipAttributes.Tier
+      AttributeNames.tier -> membershipAttributes.Tier.getOrElse("")
     ).filter(_._2.nonEmpty).map(mkAttribute[String])
 
   override def fromAttributeMap(item: collection.mutable.Map[String, AttributeValue]) =
     Attributes(
       UserId = item(AttributeNames.userId),
       MembershipNumber = item.get(AttributeNames.membershipNumber).map(_.getS),
-      Tier = item(AttributeNames.tier)
+      Tier = Some(item(AttributeNames.tier))
```

Comment:
ok, so if Tier is not set we would get an exception thrown at this point http://www.scala-lang.org/api/current/scala/collection/mutable/Map.html#apply(key:K):V